src/pages/auth/RegisterPage.jsx

```diff
@@ -2,11 +2,13 @@ import { RegisterForm } from './components/RegisterForm'
 import { gooseRegister } from '../../assets/images'
 import { GooseImage } from './components/GooseImage'
 import { AuthWrapper } from './shared.styled'
+import { ToastContainer } from 'react-toastify'
 
 const RegisterPage = () => (
   <AuthWrapper>
     <GooseImage image={gooseRegister} alt='image of goose' right={0} />
     <RegisterForm />
+    <ToastContainer />
```

Comment:
Looks like duplicate. Cound you add it one time in the root of the project. Let me know if it will not work

Reply:
done